### PR TITLE
Features/normalize distribution random

### PR DESCRIPTION
**Pull request by Plasmatium**
Since normal distribution is so common in nature, we need a  function to generate it.
`nrand` is added to calc normal distribution, with `mean` and `scale` as args.
`last-nrand` is added to refer last nrand calculated.

The `nrand` function is calculated by using [Box-Muller transform](https://en.wikipedia.org/wiki/Box%E2%80%93Muller_transform) algorithm, and code is referenced by [here](https://spin.atomicobject.com/2019/09/30/skew-normal-prng-javascript/)

### CODE REVIEW

**Comment by yuanchuan** [comment]
Thank you!

File: src/utils/random.js [hunk 1]
```diff
@@ -12,6 +12,19 @@ export default function(random) {
...
+  function nrand(mean = 0, scale = 1) {
+    console.warn(scale)
+    let u1 = 0, u2 = 0;
+    //Convert [0,1) to (0,1)
+    while (u1 === 0) u1 = Math.random();
+    while (u2 === 0) u2 = Math.random();
+    const R = Math.sqrt(-2.0 * Math.log(u1));
+    const Θ = 2.0 * Math.PI * u2;
+    const u0 = R * Math.cos(Θ);
+    console.warn('---', mean + scale * u0)
```
Comment (yuanchuan) [inline src/utils/random.js:25]: Logs need a cleanup

**Comment by Plasmatium** [comment]
Thank you for your reply.
Last three suggestions have been fixed.

Besides, `nrand-int` and some shortcuts added: 
```
nr => nrand
nri => nrand-int
```

But `nrand-int` didn't pass the test at the first time, example:
```html
    <css-doodle>
        :doodle {
            @grid: 36 / 80vmax;
            background: #12152f;
        }
        @random {
            border-left: 1px solid #5d81bc;
        }
        @random {
            border-top: 1px solid #5d81bc;
        }
        @random(.25) {
            background: linear-gradient(
                @p(#fff, tan, #5d81bc), @lp
            )
            50% / @nri(20, 10%) @lr
            /* 50% / @ri(60%) @lr */
            no-repeat;
        }
        @random {
            filter: drop-shadow(0 0 10px #fff);
        }
    </css-doodle>
```
I was expecting that `@nri` could give me like `62%, 30%, 12%, etc.`, but it didn't work with unit.
So I modified `rand-int` and `nrand-int` as below:
```javascript
    ['rand-int']({ context }) {
      return (...args) => {
        let transform_type = args.every(is_letter)
          ? by_charcode
          : by_unit;
        let rand_int = (...args) => Math.round(rand(...args))
        let value = transform_type(rand_int).apply(null, args)
        return push_stack(context, 'last_rand', value);
      }
    },
```
But I'm still not sure if it is a bug or just designed like this.

I'm using `Math.round` instead of `parseInt` in this two functions, because I'm expecting `61.8%` rounded to `62%`. If `Math.round` is not your intetion, I could commit back to `parseInt`

**Review by yuanchuan — APPROVED** [review]

**Comment by yuanchuan** [comment]
>But I'm still not sure if it is a bug or just designed like this.

Yes I think that's a bug. Thank you so much!

**Pull request merged**